openpype/plugins/publish/collect_sequence_frame_data.py

```diff
@@ -28,6 +28,10 @@ def process(self, instance):
 
     def get_frame_data_from_repre_sequence(self, instance):
         repres = instance.data.get("representations")
+        parent_entity = (
```

Comment:
To repeat myself from previous PR. This must be only `instance.data["assetEntity"]` or not filled at all.

Reply:
BTW You've also renamed `CollectFrameDataFromAssetEntity` to `CollectMissingFrameDataFromAssetEntity`, but didn't change settings name.

Reply:
Well you've changed it to use `"assetEntity"` from `instance.data`, but that is filled only after `CollectAnatomyInstanceData` which is always after this plugin (+0.49). So I guess we have to create new collector plugin which adds those data from the asset entity.

Reply:
I just realized. Why this does not happen in `CollectMissingFrameDataFromAssetEntity`? It has same family filtering and the plugin is in order which already have set `assetEntity` on instance data?